### PR TITLE
[Feature] Added Note default sort

### PR DESCRIPTION
**Pull request by yu-young-hoon**
## Description
new feature: Notes default sortBy
default-sortby

## Type of changes

- :white_circle: Bug fix (Change that fixed an issue)
- :white_circle: Breaking change (Change that can cause existing functionality to change)
- :white_circle: Improvement (Change that improves the code. Maybe performance or development improvement)
- :black_circle: Feature (Change that adds new functionality)
- :white_circle: Documentation change (Change that modifies documentation. Maybe typo fixes)

## Checklist:

- :black_circle: My code follows [the project code style](docs/code_style.md)
- :white_circle: I have written test for my code and it has been tested
- :black_circle: All existing tests have been passed
- :black_circle: I have attached a screenshot/video to visualize my change if possible

### CODE REVIEW

File: locales/en.json [hunk 1]
```diff
@@ -184,5 +184,6 @@
 	"Enable live count of notes": "Enable live count of notes",
 	"Enable smart table editor": "Enable smart table editor",
 	"Snippet Default Language": "Snippet Default Language",
-  "New notes are tagged with the filtering tags": "New notes are tagged with the filtering tags"
+  	"New notes are tagged with the filtering tags": "New notes are tagged with the filtering tags",
+  	"Default sort": "Note sort"
```
Comment (ZeroX-DG) [inline locales/en.json]: How about calling it `Default note sort`?

File: browser/main/modals/PreferencesModal/UiTab.js [hunk 2]
```diff
@@ -67,7 +67,6 @@ class UiTab extends React.Component {
       ui: {
         theme: this.refs.uiTheme.value,
         language: this.refs.uiLanguage.value,
-        defaultNote: this.refs.defaultNote.value,
```
[the remark this answers is not in the document]
Comment (yu-young-hoon) [reply browser/main/modals/PreferencesModal/UiTab.js]: my mistake ;;

File: locales/en.json [hunk 3]
```diff
@@ -184,5 +184,6 @@
 	"Enable live count of notes": "Enable live count of notes",
 	"Enable smart table editor": "Enable smart table editor",
 	"Snippet Default Language": "Snippet Default Language",
-  "New notes are tagged with the filtering tags": "New notes are tagged with the filtering tags"
+  	"New notes are tagged with the filtering tags": "New notes are tagged with the filtering tags",
```
Comment (ZeroX-DG) [inline locales/en.json]: Can you fix the indentation here?

Comment (yu-young-hoon) [reply locales/en.json]: i suspect, that folder setted sort type.
please check new folder or "window.localStorage.clear()"

i think the undefined problem is caused by my mistake commit version

**Comment by ZeroX-DG** [comment]
I've known why my problem occurred, If I pick a folder then everything works fine, but if I select the storage location instead of a folder, then the sort option won't change for the storage location (still works for folder).

**Comment by yu-young-hoon** [comment]
@ZeroX-DG 
I understood and worked on what you want.
as is
-storage, all notes, starred ... undefined: sortBy
-folder folderKey: sortBy

to be
-all notes, starred... undefined: sortBy
-storage storageKey: sortBy
-folder folderKey: sortBy

**Comment by ZeroX-DG** [comment]
@yu-young-hoon Can you resolve the conflict. Sorry for the wait.

**Comment by ZeroX-DG** [comment]
@yu-young-hoon ping

**Comment by ZeroX-DG** [comment]
Close due to inactivity

**Pull request closed without merging**